### ISSUE TITLE
Unicode support for folder and file names (HTTP 400 Bad Request)

### ISSUE THREAD

**Issue by AresEkb** [scm-manager/scm-manager#1469]
## Issue description

I'm unable to commit files or folders with non-ASCII names to SVN repositories.
I'm able to commit them to Git repositories. However I can't browse them using web-interface.

#### Describe how to reproduce the bug

Create an SVN repository. Commit folder containing non-ASCII characters in a name, for example "Компания АБВ". Get the exception:
```
Adding         Компания АБВ
svn: E175002: Commit failed (details follow):
svn: E175002: Unexpected HTTP status 400 'Bad Request' on '/scm/repo/scmadmin/company-abc-svn/!svn/wrk/a4ee4c6e-9c25-3f43-af22-74e3127e501f/%D0%9A%D0%BE%D0%BC%D0%BF%D0%B0%D0%BD%D0%B8%D1%8F%20%D0%90%D0%91%D0%92'
```

Do the same thing for the Git repository. Commit works fine. However if you'll try to open the committed folder in the web-interface, you'll get the following error: "server returned status code 400".

#### Describe your SCM-Manager setup

Just the local SCM-Manager 2.11.0 for Windows installed as a service. No plugins, additional web-servers, proxies, etc.

#### Which operating system, browser and versions do you use?

Windows 10
Chrome 87
SVN 1.14.0
Git 2.28.0

#### Log of the bug

There are no any relevant entries in the log files.

**Comment by pfeuffer** [comment]
Thanks for the report, we will have a look at it!

**Comment by pfeuffer** [comment]
Hi @AresEkb , the fix will be released with version 2.12.0 most probably this week. Thanks again for the feedback!
